src/lib/types/Augments.d.ts

```diff
@@ -34,6 +36,12 @@ declare module 'discord.js' {
 		queries: CommonQuery;
 		influx: InfluxDB | null;
 		twitch: Twitch;
+
+		emit(event: Events.GuildAnnouncementSend | Events.GuildAnnouncementEdit, message: KlasaMessage, resultMessage: KlasaMessage, channel: TextChannel, role: Role, content: string): boolean;
+		emit(event: Events.GuildAnnouncementError, message: KlasaMessage, channel: TextChannel, role: Role, content: string, error: any): boolean;
+		emit(event: Events.MoneyTransaction, target: KlasaUser, moneyToAdd: number, moneyBeforeAddition: number, action: EconomyTransactionAction, reason: EconomyTransactionReason): boolean;
```

Comment:
You can do this to avoid having to cast, same for the line below:
```suggestion
		emit(event: Events.MoneyTransaction, target: User, moneyToAdd: number, moneyBeforeAddition: number, action: EconomyTransactionAction, reason: EconomyTransactionReason): boolean;
```